ztp/argocd/src/common.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
the ztp/argocd/src is not locked to Argocd any operator has add plugin feature can execute the container or the binaries. So its better to have the directory name something like ztp/ztp-resource-generator/src or ztp/ztp-cr-generator/src.

Reply:
fixed